.github/workflows/welcome-newers.yml

```diff
@@ -0,0 +1,28 @@
+name: "Welcome new contributors"
```

Comment:
Let's change this filename to be `welcome-new-contributors`. `welcome-newers` sounds odd